<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckEditDialog</class>
 <widget class="QDialog" name="CheckEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>341</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Check</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditChkMemo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>390</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelChkMemo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Memo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTxtAmt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>710</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPayTo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>550</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumAmt">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>00000000.00; </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight</set>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelPayTo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pay to the Order of</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelChkHdg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>77</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Luxi Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Market Name
Address
City, State, Zip</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel4">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>150</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Luxi Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dollars</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOutst">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Outstanding</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonVoid">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>&amp;Void</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCleared">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Cleared</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelChkDate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Luxi Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelChkNum">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Luxi Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Check Number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3DateEdit" name="chkDate">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>50</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="order">
    <enum>Q3DateEdit::YMD</enum>
   </property>
   <property name="date">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QSpinBox" name="checkNumberSpinBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="chkSig">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>220</width>
     <height>80</height>
    </rect>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="checkTypeCombo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Miscellaneous</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Invoice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Trucker</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buyer</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3DateEdit</class>
   <extends>QWidget</extends>
   <header>Qt3Support/Q3DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkDate</tabstop>
  <tabstop>lineEditPayTo</tabstop>
  <tabstop>lineEditNumAmt</tabstop>
  <tabstop>lineEditChkMemo</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonOutst</tabstop>
  <tabstop>buttonVoid</tabstop>
  <tabstop>buttonCleared</tabstop>
  <tabstop>lineEditTxtAmt</tabstop>
 </tabstops>
 <includes>
  <include location="local">types/check.h</include>
  <include location="local">dao/checkdao.h</include>
  <include location="local">dao/invoicedao.h</include>
  <include location="local">dao/printerdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
